.github/docker_images/gcc-4.8/awslc_build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+set -ex -o pipefail
+
+echo "Building with GCC Version: $(gcc --version)"
+
+BUILD_DIR=$(mktemp -d)
+SRC_DIR="${SRC_DIR:-/awslc}"
+
+pushd "${BUILD_DIR}"
+
+cmake "${SRC_DIR}" "-DDISABLE_GO=ON" "-DDISABLE_PERL=ON" "-DMY_ASSEMBLER_IS_TOO_OLD_FOR_512AVX=1"
```

Comment:
Do we want to test also with Perl and Go available, or it's enough we're doing that in our CI?

Reply:
Right, we already have several tests in our CI that test GCC 4.8 with Go and Perl enabled. The focus of this test is to verify that we can build with GCC 4.8 and the pre-generated assembly. (I'm not sure that there is a Go version able to run our Go code that is compatible with the glibc version available on that docker image.)